jobs/pull_requests.rb

```diff
@@ -32,7 +32,7 @@
 	trend_class = GithubDashing::Helper.trend_class(trend)
 
 	send_event(
-		'pull_requests', 
+		'pull_requests',
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.